extensions/youtube-downloader/src/index.tsx

```diff
@@ -167,9 +179,15 @@ export default function DownloadVideo() {
   }, [video]);
 
   useEffect(() => {
+    // why nested promise? because we prioritize clipboard over active tab url
```

Comment:
Let's remove this useless comment.

In the meantime, you can wrap this code in another async function to avoid nested promise-then.